packages/components/src/Select/ListBoxSection.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useListBoxSection } from '@react-aria/listbox';
+import type { ListState } from '@react-stately/list';
+import { useStyles } from '@marigold/system';
+import type { Node } from '@react-types/shared';
+
+import { Box } from '../Box';
+import { Option } from './Option';
+
+interface SectionProps {
+  section: Node<unknown>;
+  state: ListState<unknown>;
```

Comment:
I guess the `<unknown>` type causes the problem in Storybook that we can't import and use `Item` as intended.